infrastructure/hosts/catcolab-next/backend.nix

```diff
@@ -194,6 +221,16 @@ in {
         ]; 
     }];
 
+
+    users.users.deployuser = {
+        isNormalUser = true;
+        openssh.authorizedKeys.keys = [
+            ''
+                command="${lib.getExe updateScript}",no-port-forwarding,no-agent-forwarding,no-X11-forwarding,no-pty ${catcolabnextDeployuserKey}
```

Comment:
Declaration and use of the existing scripts is strange blend of manual unixy install steps and declarative nixy configuration. Mixing them seems wrong so I added the new one the nix way. I'm not sure if this was the right call.

Reply:
Again, I'm not the right person to advise here, but if you see other ways to make the Nix config more idiomatic and/or declarative, please go ahead and do so!